receiver/hostmetricsreceiver/internal/scraper/processscraper/config.go

```diff
@@ -43,6 +43,10 @@ type Config struct {
 	// doesn't exist on the system, eg. is owned by user existing in container only
 	MuteProcessUserError bool `mapstructure:"mute_process_user_error,omitempty"`
 
+	// MuteProcessAllErrors is a flag that will mute all the errors encountered when trying to read metrics of a process
+	// the collector does not have permission to read.
+	MuteProcessAllErrors bool `mapstructure:"mute_process_all_errors,omitempty"`
```

Comment:
A note about this should be added to the README. We'll want to include information about how this interacts with the other `mute_*_error` configuration options, essentially how precedence works for these overlapping options.